### PR TITLE
add vergecurrency support

### PR DESCRIPTION
**Pull request by justinvforvendetta**

### CODE REVIEW

File: install_wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
```
Comment (barthalion) [inline install_wrapper.sh]: Is there a reason why it's not statically done via `simple` buildsystem?

File: org.vergecurrency.verge-qt.desktop [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
```
Comment (barthalion) [inline org.vergecurrency.verge-qt.desktop]: Ideally should be shipped in your tarball/archive for others to consume.

File: org.vergecurrency.verge-qt.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline org.vergecurrency.verge-qt.appdata.xml]: Ideally should be shipped in your tarball/archive for others to consume.

File: org.vergecurrency.verge-qt.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
+{
+  "id": "org.vergecurrency.verge-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "6.0.2",
```
Comment (barthalion) [inline org.vergecurrency.verge-qt.json]: This is the version of runtime specified in `runtime`. You probably want to set it to `19.08`.

File: org.vergecurrency.verge-qt.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        "install -m 0744 -D -t /app/bin/internal bin/verged",
+        "install -m 0744 -D -t /app/bin/internal bin/verge-qt",
+        "install -m 0744 -D -t /app/bin/internal bin/verge-cli",
+        "install -m 0744 -D -t /app/bin/internal bin/test_verge"
+      ],
+      "post-install": ["/app/bin/internal/test_verge"],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
```
Comment (barthalion) [inline org.vergecurrency.verge-qt.json]: if you don't want to build it from source, create `flathub.json` with the same key to limit the build to x86_64.

File: org.vergecurrency.verge-qt.json (hunk 4 above)
Comment (justinvforvendetta) [reply org.vergecurrency.verge-qt.json]: resolved

File: install_wrapper.sh (hunk 1 above)
Comment (justinvforvendetta) [reply install_wrapper.sh]: this is a wrapper for the blockchain data directory

File: org.vergecurrency.verge-qt.desktop (hunk 2 above)
Comment (justinvforvendetta) [reply org.vergecurrency.verge-qt.desktop]: we can certainly add that to the tar.

File: org.vergecurrency.verge-qt.json (hunk 5 above)
Comment (justinvforvendetta) [reply org.vergecurrency.verge-qt.json]: ah good suggestion, i didn't see this in my references. thanks!

File: org.vergecurrency.verge-qt.appdata.xml (hunk 3 above)
Comment (justinvforvendetta) [reply org.vergecurrency.verge-qt.appdata.xml]: will look into that, certainly seems feasible.

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt.json

**Comment by justinvforvendetta** [comment]
well, runtime ver, and only x86 seem to work. not sure why it added an extra ".json" to the end of the .json

**Comment by barthalion** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
I see, I will add the checksum for the .svg

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
@barthalion can  you think of any reason why it keeps rejecting my sha256 hash of the vector image?

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
@barthalion finally! lol it's great you guys have a build bot that anyone can use for testing. makes integration 100x easier.

**Comment by bilelmoussaoui** [comment]
Anyone else from upstream that might need write access to the repository once created?

File: org.vergecurrency.verge-qt.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        "bash install_wrappers.sh"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "org.vergecurrency.verge-qt.desktop"
+        },
+        {
+          "type": "file",
+          "path": "org.vergecurrency.verge-qt.appdata.xml"
```
Comment (LeandroStanger) [inline org.vergecurrency.verge-qt.json]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by justinvforvendetta** [comment]
> Anyone else from upstream that might need write access to the repository once created?

@marpme and myself

**Comment by justinvforvendetta** [comment]
bot, build org.vergecurrency.verge-qt

**Comment by justinvforvendetta** [comment]
@barthalion @bilelmoussaoui thanks guys!

**Comment by justinvforvendetta** [comment]
@barthalion i think you forgot to add me as a contributor as well. seems only marpme is right now. thanks for everything! cheers =]

**Pull request closed without merging**